servers/src/test/java/tachyon/master/EditLogTest.java

```diff
@@ -0,0 +1,250 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.master;
+
+import java.io.DataInputStream;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.commons.codec.binary.Base64;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.type.TypeReference;
+
+import tachyon.TachyonURI;
+import tachyon.conf.TachyonConf;
+
+/**
+ * Unit Test for EditLog. Test the read/write correctness of each operation.
+ */
+public class EditLogTest {
+  private static final long TEST_OP_TIME_MS = 1409349750338L;
+  private static final long TEST_TRANSACTION_ID = 100L;
+
+  private EditLog mEditLog = null;
+  private String mEditLogPath = null;
+  private TachyonConf mTachyonConf = new TachyonConf();
+
+  @Rule
+  public TemporaryFolder mTestFolder = new TemporaryFolder();
+
+  @Before
+  public final void before() throws Exception {
+    mEditLogPath = mTestFolder.newFile().getAbsolutePath();
+    mEditLog = new EditLog(mEditLogPath, false, TEST_TRANSACTION_ID, mTachyonConf);
+  }
+
+  /**
+   * A util method to load an EditLogOperation from the edit log file. In this test, each edit log
+   * file has exact one operation.
+   * @param path the path of edit log file
+   * @return the loaded operation
+   * @throws IOException
+   */
+  private EditLogOperation getSingleOpFromFile(String path) throws IOException {
+    DataInputStream is = new DataInputStream(new FileInputStream(path));
+    JsonParser parser = JsonObject.createObjectMapper().getFactory().createParser(is);
+    EditLogOperation ret = parser.readValueAs(EditLogOperation.class);
+    is.close();
+    return ret;
+  }
+
+  @Test
+  public void addBlockTest() throws IOException {
+    // Write ADD_BLOCK
+    mEditLog.addBlock(1, 2, 128L, TEST_OP_TIME_MS);
```

Comment:
Some of these can also be constants. If they are never reused in a different test, you can at least create local variable to use for the test asserts.